src/main/java/org/isf/menu/manager/UserBrowsingManager.java

```diff
@@ -97,9 +99,13 @@ public User getUserByName(String userName) throws OHServiceException {
 	 */
 	public User newUser(User user) throws OHServiceException {
 		String username = user.getUserName();
+		if (!username.matches(VALID_USERID_PATTERN)) {
+			throw new OHDataValidationException(
+							new OHExceptionMessage(MessageBundle.getMessage("angal.userbrowser.theusernamecontainsinvalidcharacters.msg")));
+		}
```

Comment:
why we don't adopt the "validation" pattern like in the other managers?

Reply:
Because this is the one and only place it is needed to be checked.

Reply:
I see